packages/portfolio-contract/test/snapshots/portfolio.flows.test.ts.md

```diff
@@ -293,6 +293,9 @@ Generated by [AVA](https://avajs.dev).
               value: 50000000n,
             },
             dest: 'USDNVault',
+            detail: {
+              usdnOut: 49500000n,
+            },
             how: 'USDN',
             src: '@noble',
```

Comment:
Nit: I don't think it's worth making a change here, but since AVA and browser DevTools and a large handful of other software present object fields in alphabetical order, `src` and `target` tend to make a better pair than `src` and `dest` (which are not only in an unhelpful `dest`-first order, but are separated by intervening fields). I'll be keeping this in mind going forward, and may even rename accordingly in the network and LP solver data models.

Reply:
"target" gets used for "target allocation", so that pair would be surprising and potentially confusing.

Reply:
Fwiw vstorage is abstracted from the UI so if we do want to change we just have to in vstorage, and it could be made to support both schemas.